### ISSUE TITLE
First install on macOS 10.14.2 - just sits there

### ISSUE THREAD

**Issue by raguay** [ipfs-shipyard/peer-pad#281]
I just found this today and tried to install it. After starting it with the `npm start` it says it's serving the page and gives the addresses, but the page in Chrome is only blank. It loads for a very long time and then just blank.

I have the ipfs command line installed with ipfs-install, the macOS menu application, and the Chrome extension for IPFS. They were all running at the time. Is that an issue?

**Comment by jimpick** [comment]
It does take a little time the first time you load the page in development mode, as it is a react app and it has to "compile" and "bundle" all the frontend files together (using webpack). When I start it on my MacBook, it takes about 30-60 seconds, and then it says:

```
Compiled successfully!

You can now view peerpad in the browser.

  Local:            http://localhost:3000/
  On Your Network:  http://10.0.1.11:3000/

Note that the development build is not optimized.
To create a production build, use npm run build.
```

It automatically launches the web page in the default web browser once that shows on the terminal.

If it doesn't show that information in the terminal window, I'm guessing the compilation is stuck somehow?

I'm OS X Mojave 10.14.2 as well. Running `node -v` says my node.js version is v10.13.0.

It shouldn't impact whether the page loads or not, but you also should run `npm run start:rendezvous` in a separate terminal window when you are doing local development.

**Comment by raguay** [comment]
Yes, It says all those things, but the webpage never loads. My node.js version is 11.6.

Okay. It just started running, but it gives this error a lot:
```
Unhandled Rejection (TypeError): Cannot read property 'sub' of null
```

**Comment by jimpick** [comment]
Hmmm. I just tried node 11.6.0 here, and it compiled and ran. I managed to run it in Chrome, Firefox Nightly and Safari. I also tried it with IPFS Companion turned on (in Chrome).

I'm having some trouble reproducing the issue. Another source of variation might be related to the various native modules that get compiled when doing `npm install`.
